scripts/pylint_extensions.py

```diff
@@ -122,10 +114,7 @@ def visit_call(self, node):
                     try:
                         func_name = node.func.attrname
                     except AttributeError:
-                        try:
-                            func_name = node.func.name
-                        except AttributeError:
-                            func_name = node.func
+                        func_name = node.func.name
```

Comment:
Since this is a call node, it will always point to a function, hence node will always have an attribute func